app/Models/Role.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Role extends Model
+{
+    protected $fillable = [
+        'name',
+        'description'
```

Comment:
```suggestion
        'description',
```
c cài phpcs rồi chạy thử sẽ thấy báo lỗi nhé